src/studio/src/designer/backend/Services/Interfaces/IGitRepository.cs

```diff
@@ -0,0 +1,9 @@
+namespace Altinn.Studio.Designer.Services.Interfaces
+{
+    /// <summary>
+    /// Interface for handling files in a Git repository.
+    /// </summary>
+    public interface IGitRepository
```

Comment:
not sure how I feel about this empty interface and class.. Why not create them once ready to add functionality to them?

Reply:
Oh, I thought I had left them out. I'll remove them and bring them back once they're needed.